odoo/addons/base/models/res_partner.py

```diff
@@ -239,6 +239,25 @@ def default_get(self, default_fields):
         ('check_name', "CHECK( (type='contact' AND name IS NOT NULL) or (type!='contact') )", 'Contacts require a name'),
     ]
 
+    def _compute_avatar_1920(self):
+        return self._compute_avatar_xxxx('1920')
+
+    def _compute_avatar_128(self):
+        return self._compute_avatar_xxxx('128')
```

Comment:
These overrides should have extra dependencies, for all the values that are used on it.
Also for the correctness of the code, all sizes should have the override, not only those that are used right now.